### PR TITLE
Change "remove tags" to "hide tags"

### PR DESCRIPTION
**Pull request by Kazephil**
Replace "remove tags" with "hide tags" in the UI, and update the appropriate descriptions in the manual.

### CODE REVIEW

**Comment by brandelune** [comment]
`... will be hidden and you won't have to manipulate them.`

could be a nice addition.

**Comment by Kazephil** [comment]
> `... will be hidden and you won't have to manipulate them.`

could be a nice addition. 

@brandelune
I just rephrased the entire sentence to include that idea.
Have a look and let me know what you think.

**Review by brandelune — CHANGES_REQUESTED** [review]

File: doc_src/en/Dialogs_ProjectProperties.xml [hunk 1]
```diff
@@ -199,24 +199,23 @@
...
 		  
 		  <para>This is especially useful when dealing with texts where inline
 		  formatting is not particularly useful (e.g., OCRed PDF, or poorly
 		  converted .odt or .docx files.)</para>
 
 		  <note>
-			<para>The tags are not really removed from the document, but rather
-			stacked at the end of the segment. In most cases, this should not
+			<para>Hidden tags are simply stacked at the end of the segment. In most cases, this should not
 			keep the translated file from opening. If you just need to reduce
```
[the remark this answers is not in the document]
Comment (brandelune) [reply doc_src/en/Dialogs_ProjectProperties.xml:230]: I think you should see it now.

**Comment by Kazephil** [comment]
@brandelune

I think my latest commit covers everything in your comments.

**Pull request merged**